src/index.html

```diff
@@ -9,10 +9,41 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <div>
+        <img src="images/imac.jpeg" alt="imacPh">
+      </div>
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="card__info">
+        Product code: 195434
+      </p>
+      <div class="rewiews"></div>
```

Comment:
The class name 'rewiews' is likely a typo. It should be 'reviews' to maintain consistency and avoid confusion.